include/arch/ia32/ia32.h

```diff
@@ -73,7 +73,10 @@ typedef struct {
 
 typedef struct {
 	enum { pctl_set = 0, pctl_get } action;
-	enum { pctl_pci = 0, pctl_busmaster, pctl_reboot } type;
+	enum { pctl_pci = 0,
+		pctl_busmaster,
+		pctl_reboot,
+		pctl_graphmode } type;
```

Comment:
I know that the clang-format complains, but we'd like to keep the formatting (at least for now) as it was

Reply:
Sure, corrected